sources/packages/forms/src/form-definitions/noticeofassessment.json

```diff
@@ -4995,12 +4965,538 @@
+                            "disabled": true
+                          }
+                        ]
+                      },
+                      {
+                        "components": [
+                          {
+                            "autofocus": false,
+                            "input": true,
+                            "tableView": false,
+                            "inputType": "text",
+                            "inputMask": "",
+                            "label": "Text",
+                            "key": "table5Text16",
+                            "placeholder": "",
+                            "prefix": "",
+                            "suffix": "",
+                            "multiple": false,
+                            "defaultValue": "",
+                            "protected": false,
+                            "unique": false,
+                            "persistent": false,
+                            "hidden": false,
+                            "clearOnHide": false,
+                            "spellcheck": false,
+                            "validate": {
+                              "required": false,
+                              "minLength": "",
+                              "maxLength": "",
+                              "pattern": "",
+                              "custom": "",
+                              "customPrivate": false
+                            },
+                            "conditional": {
+                              "show": "",
+                              "when": null,
+                              "eq": ""
+                            },
+                            "type": "textfield",
+                            "labelPosition": "top",
+                            "inputFormat": "html",
+                            "tags": [],
+                            "properties": {},
+                            "hideLabel": true,
+                            "calculateValue": "value = typeof data.assessment?.parentalDiscretionaryContribution === 'number' ? '$' + (data.assessment.parentalDiscretionaryContribution).toLocaleString('en-CA', { minimumFractionDigits: 2 }) : '-';",
+                            "disabled": true,
+                            "isNew": false
+                          }
+                        ]
+                      }
+                    ],
+                    [
+                      {
+                        "components": [
+                          {
+                            "key": "table5Content8",
+                            "label": "Parental asset contribution",
+                            "input": false,
+                            "tag": "p",
+                            "attrs": [
+                              {
+                                "value": "",
+                                "attr": ""
+                              }
+                            ],
+                            "className": "ml-4",
```

Comment:
Can we create a shared style like the below in `formio-shared.scss`?  It would allow changing the style for all of them at once and maybe apply the same fro other grid rows.
```scss
.grid-row-indentation-first-level {
  margin-left: 1rem;
}
````